src/App.js

```diff
@@ -7,74 +7,116 @@ import ListOfMembers from "./components/ListOfMembers";
 import ListOfTrainers from "./components/ListOfTrainers";
 import CreateMembers from "./Pages/CreateMembers";
 import ViewMember from "./Pages/ViewMemeber"
-import EditMember from "./Pages/EditMember";
+import EditMember from "./Pages/EditMemberForm";
```

Comment:
Although this is fine, the name of the component in the file is `EditMemberForm` - I'd recommend changing this to `import EditMemberForm` just for clarity with team members who might come to use it